editoast/src/views/path/properties.rs

```diff
@@ -275,16 +208,52 @@ async fn cache_path_properties(
 }
 
 /// Compute path properties input hash without supported electrifications
-fn path_properties_input_hash(
-    infra: i64,
-    infra_version: i64,
-    track_ranges: &[TrackRange],
-) -> String {
+fn path_properties_input_hash(path_properties_request: &PathPropertiesRequest<'_>) -> String {
     let osrd_version = get_app_version().unwrap_or_default();
     let mut hasher = DefaultHasher::new();
-    track_ranges.hash(&mut hasher);
+    path_properties_request
+        .track_section_ranges
+        .hash(&mut hasher);
     let hash_track_ranges = hasher.finish();
-    format!("path_properties.{osrd_version}.{infra}.{infra_version}.{hash_track_ranges}")
+    format!(
+        "path_properties.{osrd_version}.{}.{}.{hash_track_ranges}",
+        path_properties_request.infra, path_properties_request.expected_version
+    )
+}
+
+async fn compute_path_properties_batch(
+    core_client: Arc<CoreClient>,
+    valkey_conn: &mut ValkeyConnection,
+    path_properties_requests: &[PathPropertiesRequest<'_>],
+) -> Result<Vec<PathProperties>> {
+    let mut path_properties_result = Vec::with_capacity(path_properties_requests.len());
+
+    for request in path_properties_requests {
+        match retrieve_path_properties_from_cache(valkey_conn, request).await? {
+            Some(path_properties) => {
+                info!("Hit cache");
```

Comment:
I'm not sure that we want to log that each time we find an entry in the cache. At least if we do, maybe `debug` is more appropriate.